lib/bin/check-file-headers.js

```diff
@@ -48,11 +50,21 @@ process.stdin.on('end', () => {
     if (!withConsistentYear.startsWith(header)) {
       console.error(entryPath);
       missing = true;
+
+      if (mode === 'fix') {
+        fs.writeFileSync(entryPath, `${header}\n${contents}`);
```

Comment:
I think it'd be good to add a file header with the current year (rather than 2017).